dashboard/cron.py

```diff
@@ -358,26 +389,34 @@ def do(self):
 
         status += "Start cron at: " +  str(datetime.datetime.now()) + "\n"
 
-        logger.info("************ term")
-        status += self.update_with_udw_term()
+        invalid_course_id_list = self.invalid_course_ids()
+        logger.debug(f"invalid id {invalid_course_id_list}")
+        if (len(invalid_course_id_list) > 0):
+            # error out and stop cron job
+            status += f"ERROR: Those course ids are invalid: {invalid_course_id_list}\n"
```

Comment:
Rather than indenting all of this in an else you could have just 

`return status`

here.